nucliadb_node/src/replication/replicator.rs

```diff
@@ -267,6 +267,10 @@ pub async fn connect_to_primary_and_replicate(
                     Some(shard_state.kbid.clone()),
                     shard_state.similarity.clone().into(),
                     None,
+                    // read replicas won't have to index resources, they don't
+                    // need this parameter for anything, so we avoid passing it
+                    // through the pb
+                    false,
```

Comment:
Well, they don't need to index resources, but they need to query them. And the search vector should be normalized in the same way, so we need to know at search time if we need to normalize. So we actually need this.

The metadata.json file is not sent directly through replication, but rather created from this metadata.

Reply:
Ups, that's absolutely true! Thanks